src/providers/treeViewProvider.ts

```diff
@@ -0,0 +1,63 @@
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+export class HttpTreeProvider implements vscode.TreeDataProvider<HttpClientItem> {
+
+    private _onDidChangeTreeData: vscode.EventEmitter<HttpClientItem | undefined> = new vscode.EventEmitter<HttpClientItem | undefined>();
+    readonly onDidChangeTreeData: vscode.Event<HttpClientItem | undefined> = this._onDidChangeTreeData.event;
+
+    constructor() {
+    }
+
+    refresh(): void {
+        this._onDidChangeTreeData.fire();
+    }
+
+    getTreeItem(element: HttpClientItem): vscode.TreeItem {
+        return element;
+    }
+
+    getChildren(): Thenable<HttpClientItem[]> {
+        return Promise.resolve(this.getDepsInPackageJson());
+    }
+
+    private getDepsInPackageJson(): Thenable<HttpClientItem[]> {
```

Comment:
A better name, this seems not related to our code

Reply:
changed it.